Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -154,6 +156,19 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
     static constexpr int invalidContextMenuItemTag { -1 };
 #endif
 
+    enum class SelectionGranularity : uint8_t {
+        Character,
+        Word,
+        Line,
+    };
+    enum class CommitReason : bool { SelectionIsNoLongerActive, ReceivedMouseUp };
```

Comment:
UnifiedPDFPlugin::CommitReason : not obvious that it's about selection

Reply:
Renaming to `SelectionCommitReason`

Reply:
Renamed in [53cae5f](https://github.com/WebKit/WebKit/commit/53cae5fe11eca85d9ea822cdec0181e4a4f4dcf6)